### ISSUE TITLE
Warning when running any shell commands

### ISSUE THREAD

**Issue by ph** [elastic/logstash#3087]
Reproduced on a Windows Server 2012 R2 and Ubuntu

When executing any logstash command I have this warning:
`io/console not supported; tty will not be manipulated`

This message doesn't impact the execution of any command.

---

Possibly a jruby issue and fixed in .20?
https://github.com/jruby/jruby/issues/1614#issuecomment-94798150

I dot not recall having this warning on the previous RCs when doing the smoke testing on windows.

**Comment by ph** [comment]
@jsvd Did you get this error when doing the smoke testing on windows?

**Comment by jsvd** [comment]
yes always remember seeing that on windows

**Comment by suyograo** [comment]
I see this on ubuntu too, noticed it after JRuby upgrade to 1.7.19

**Comment by ph** [comment]
Updated to include ubuntu.

**Comment by boskowski** [comment]
```
C:\>jruby -v -e ""
jruby 1.7.20 (1.9.3p551) 2015-05-04 3086e6a on Java HotSpot(TM) 64-Bit Server VM 1.8.0_31-b13 +jit [Windows 7-amd64]
io/console not supported; tty will not be manipulated
```

**Comment by ph** [comment]
This has been fixed in the jruby-1.7.24 releases, I haven't seen that issue on ubuntu either.
